Material.Styles/Resources/Themes/Expander.axaml

```diff
@@ -99,6 +103,7 @@
   </ControlTheme>
 
   <ControlTheme x:Key="MaterialExpander" TargetType="Expander">
+    <Setter Property="TextBlock.TextAlignment" Value="Center" />
```

Comment:
I suggest leave default value (TextAlignment.Start) for this, since i think what by default text should be in left corner as described in [specs](https://m1.material.io/components/expansion-panels.html#expansion-panels-behavior).

Reply:
agree